Classes/Controller/BackendController.php

```diff
@@ -118,7 +118,7 @@ public function buildFormAction($view = 'BuildForm') {
 	 * @return void
 	 */
 	public function buildAction($name, $author, $title, $description, $controllers, $pages, $content, $backend, $vhs, $dry, $verbose) {
-		$generator = $this->extensionService->buildProviderExtensionGenerator($name, $author, $title, $description, $controllers, $pages, $content, $backend, $vhs, $dry, $verbose);
+		$generator = $this->extensionService->buildProviderExtensionGenerator($name, $author, $title, $description, $controllers, $pages, $content, $backend, $vhs);
```

Comment:
Both arguments are not supported and will be set in next 2 lines via setter.
https://github.com/FluidTYPO3/builder/pull/69/files#diff-2c8d468cf2d54911b45ca38d4787091dL122